app/views/admin/banners/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<% @page_header = "Active Banners" %>
+
+
+<table id="search-results-list">
+
+  <tr>
+    <th style="width: 20%">Expires at</th>
+    <th>Message</th>
+    <th>Edit</th>
+    <th>Delete</th>
+  </tr>
+
+  <% @banners.each do |banner| %>
+    <% zebra = cycle('odd', 'even') %>
+    <tr class="<%= zebra %>">
+      <td><%= banner.expires_at.in_time_zone('UTC').strftime("%m/%d/%Y %I:%M%p") %></td>
```

Comment:
@Dantemss I don't know why this works this way, seems counterintuitive to me, but it does seem to work fine. I'd consider moving this timezone logic and `strftime` display logic somewhere else but not sure where or if it's really necessary.

Reply:
Right... that's right. Like I said, I don't know why this works this way but it looks right in the frontend. @Dantemss

Reply:
Hm can you move this code to the banner model (`def expires_at`) and write a few tests? Just to be safe.

Reply:
Actually maybe call it something else since it returns a string.